test/evm_script.js

```diff
@@ -93,9 +93,16 @@ contract('EVM Script', accounts => {
                 const action = { to: executionTarget.address, calldata: executionTarget.contract.execute.getData() }
                 const script = encodeCallScript([action])
 
-                await executor.execute(script)
+                const receipt = await executor.execute(script)
 
                 assert.equal(await executionTarget.counter(), 1, 'should have executed action')
+
+                // Check logs
+                // The Executor always uses 0x for the input and callscripts always have 0x returns
+                const scriptResult = receipt.logs.filter(l => l.event == 'ScriptResult')[0]
+                assert.equal(scriptResult.args.script, script)
```

Comment:
I guess we could add some descriptions to these assertions, like for most of the rest of them, to help in case it fails.